features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/utils/SemanticsUtils.kt

```diff
@@ -212,4 +239,45 @@ internal class SemanticsUtils {
             fontFamily = layoutInput.style.fontFamily
         )
     }
+
+    private fun resolveOnDrawInstance(semanticsNode: SemanticsNode): Any? {
+        val drawBehindElement =
+            semanticsNode.layoutInfo.getModifierInfo().firstOrNull { modifierInfo ->
+                ComposeReflection.DrawBehindElementClass?.isInstance(modifierInfo.modifier) == true
+            }?.modifier
+
+        return drawBehindElement?.let {
+            ComposeReflection.OnDrawField?.getSafe(drawBehindElement)
+        }
+    }
+
+    private fun resolveReflectedProperty(semanticsNode: SemanticsNode, fieldType: CheckmarkFieldType): Long? {
+        val onDrawInstance = resolveOnDrawInstance(semanticsNode)
+
+        val checkmarkColor: AnimationState<*, *>? = onDrawInstance?.let {
+            when (fieldType) {
+                CheckmarkFieldType.FILL_COLOR -> {
+                    ComposeReflection.BoxColorField?.getSafe(onDrawInstance) as? AnimationState<*, *>
+                }
+                CheckmarkFieldType.CHECKMARK_COLOR -> {
+                    ComposeReflection.CheckColorField?.getSafe(onDrawInstance) as? AnimationState<*, *>
+                }
+                CheckmarkFieldType.BORDER_COLOR -> {
+                    ComposeReflection.BorderColorField?.getSafe(onDrawInstance) as? AnimationState<*, *>
+                }
+            }
+        }
+
+        val result = (checkmarkColor?.value as? androidx.compose.ui.graphics.Color)?.value
```

Comment:
seems like can be moved to `import`? I don't see any `Color` listed there so far.